geco/mips/miplib/loader.py

```diff
@@ -0,0 +1,71 @@
+import tempfile
+import requests
+import pyscipopt as scip
+import os
+import pandas as pd
+
+INSTANCES_DIR = tempfile.gettempdir() + "/geco/miplib/instances/"
```

Comment:
If I'm seeing this correctly, the tempdir does not get deleted by python.
I think we could make use of a class as a MIPLIB loader and add a destructor to it, that deltes the tempfile (and all its contents). As we are caching anyways, having a statefull class seems reasonable. What do you think?

Reply:
Just a suggestion (to have all the options before making a decision), we can avoid using a class by adding a clear_cache() function, this would work right?

Reply:
This would also work. I just think it's unexpected from a users perspective to have files left on the disk after the program finishes (even if they are placed in /tmp).

Reply:
I changed the implementation to use temp files, they get deleted when the program finishes execution.

Reply:
Is there still an advantage to using a class to this implementation?